src/inlineeditoruiview.js

```diff
@@ -126,24 +134,40 @@ export default class InlineEditorUIView extends EditorUIView {
 // @private
 // @type {module:utils/dom/position~Options#positions}
 const panelPositions = [
-	( editableRect, panelRect ) => ( {
-		top: editableRect.top - panelRect.height,
-		left: editableRect.left,
-		name: 'toolbar_nw'
-	} ),
-	( editableRect ) => ( {
-		top: editableRect.bottom,
-		left: editableRect.left,
-		name: 'toolbar_sw'
-	} ),
-	( editableRect, panelRect ) => ( {
-		top: editableRect.top - panelRect.height,
-		left: editableRect.left + editableRect.width - panelRect.width,
-		name: 'toolbar_ne'
-	} ),
-	( editableRect, panelRect ) => ( {
-		top: editableRect.bottom,
-		left: editableRect.left + editableRect.width - panelRect.width,
-		name: 'toolbar_se'
-	} )
+	( editableRect, panelRect ) => {
+		return {
+			top: getPanelPositionTop( editableRect, panelRect ),
+			left: editableRect.left,
+			name: 'toolbar_west'
+		};
+	},
+	( editableRect, panelRect ) => {
+		return {
+			top: getPanelPositionTop( editableRect, panelRect ),
+			left: editableRect.left + editableRect.width - panelRect.width,
+			name: 'toolbar_east'
```

Comment:
They are directly translated into `.ck-balloon-panel_toolbar_east` CSS classes. They should follow BEM.